audit/projects/kubernetes-public/secrets/k8s-infra-prow-cookie/description.json

```diff
@@ -0,0 +1,7 @@
+{
+  "createTime": "2021-08-04T22:01:54.096956Z",
+  "name": "projects/127754664067/secrets/k8s-infra-prow-cookie",
```

Comment:
I'm going to assume this is @ameukam as part of https://github.com/kubernetes/k8s.io/pull/2235